app/components/c-modal-header-01.vue

```diff
@@ -64,18 +65,14 @@
 
       /**
        * Sets a modifier class for title spacing options
+       *
+       * Valid values: [0, 300, 500]
```

Comment:
- [ ] I think we should move the valid values to the top of the comment, so it's easier to see in the styleguidist. As it's now this will create a new line in the props documentation. That could also be a valid solution but I think we should unify this then. What's your opinion on that?